src/main/java/org/wirvsvirus/locoronando/location/GeocodingServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package org.wirvsvirus.locoronando.location;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import org.wirvsvirus.locoronando.dealer.Address;
+
+@Service
+public class GeocodingServiceImpl implements GeocodingService {
+
+	private static final String ZIPCODE_SEARCH_STRING = "%s Germany";
```

Comment:
Why do constants always have to be on top? It would be easier to read if you put this directly above the method.

Reply:
We probably should even move the value to the application.properties.

Reply:
moved to properties